scripts/packTranslations.js

```diff
@@ -0,0 +1,169 @@
+/**
```

Comment:
Instead of manipulating string tokens, an alternative would be:

```js
let finalJSON = {};
for each file:
    const rawString = fs.readFileSync(file.json, 'utf8')
    const json = JSON.parse(rawString);
    finalJson[fileName] = json

fs.writeFileSync(JSON.stringify(finalJSON))
```

Just an idea, no need to change the current script if it works, as you said it this is good enough.

Reply:
Sorry, somehow I miss this review (funny though, I shouldn't miss this). I thought about use JSON too, but I tried to make the script processes as fast as possible. Therefore, I decide to load and shift every bit from buffer with stream. I also don't know what will happen if we uses JSON method on a lot of namespace files.

However, recently I found out a new problem when encounter this in another aspect. What if we need to update old file. Appearently string tokens would fail. So maybe I will change to JSON or find other methods for this. Right now, I would like to focus on complete/improve i18next implement process.